### ISSUE TITLE
API calls dependency

### ISSUE THREAD

**Issue by william-second** [cuckoosandbox/cuckoo#1365]
is there any way to obtain API call dependency between two function in Cuckoo? In other word, can we determine whether the output of API call X will be used as an input for API call Y? 
Thank you.

**Comment by jbremer** [comment]
No, Cuckoo doesn't provide taint information itself, although you can of course create a pintool analysis module and do something like that yourself if you really want to. Does this answer your question?

**Comment by william-second** [comment]
Hi Jbremer, indeed I am now want to create a pintool analysis module to do this. May I ask if you know how to do it, or do you know a pointer so that I can take it as a starting point? 
Thank you very much,
